technic/machines/network.lua

```diff
@@ -132,7 +137,7 @@ local function touch_node(tier, pos, timeout)
 		-- this should get built up during registration
 		node_timeout[tier] = {}
 	end
-	node_timeout[tier][poshash(pos)] = timeout or 5
+	node_timeout[tier][poshash(pos)] = timeout or default_timeout
```

Comment:
Changed again to be dynamic based on current throttling interval rounded up + 1.